src/run/invoke_run.jl

```diff
@@ -86,31 +82,34 @@ function default_save_name(rs::ADRIA.ResultSet)::String
     return "$(rs.name)__RCPs_$(rs.RCP)__$(rs.invoke_time)"
 end
```

Comment:
Noting here that the docstring here is incorrect.

A user can specify their own results directory via the `.config.toml` file, as documented [here](https://open-aims.github.io/ADRIA.jl/dev/usage/getting_started/#Setup)

Hypothetically, you could override the config programmatically as well.

Reply:
Actually, not sure we need this method?

You can just do: `ADRIA.result_location(rs)`

https://github.com/open-AIMS/ADRIA.jl/blob/a2c63755324a175472043e9f9a00c46dba8feeb8/docs/src/usage/cookbook.md#L106